WooCommerce/Classes/ViewRelated/Dashboard/DashboardViewModel.swift

```diff
@@ -153,6 +155,9 @@ final class DashboardViewModel: ObservableObject {
             group.addTask { [weak self] in
                 await self?.topPerformersViewModel.reloadData()
             }
+            group.addTask { [weak self] in
+                await self?.mostActiveCouponsViewModel.reloadData()
+            }
```

Comment:
We should hide this behind the M2 feature flag to avoid fetching unnecessary data.

Reply:
Good point. Done in 26bbf3597dd89242483cd5e5b0b29f7bd8aad71d